config/routes/candidate.rb

```diff
@@ -20,8 +20,9 @@
 
   resources :pool_opt_ins, only: %i[new create edit update], path: 'preferences-opt-in'
   resources :draft_preferences, only: %i[show update], path: 'preferences' do
-    resource :publish_preferences, only: %i[show create], path: 'publish-preferences'
+    resources :training_locations, only: %i[new create], path: 'training-locations'
     resources :location_preferences, path: 'location-preferences'
+    resource :publish_preferences, only: %i[show create], path: 'publish-preferences'
```

Comment:
This is just cosmetic, I put them in the order they appear on the page for my own clarification.